fairlearn/linear_model/__init__.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) Fairlearn contributors.
+# Licensed under the MIT License.
+
+"""This module contains linear models adapted to adhere to fairness constraints."""
```

Comment:
This file is not tested because you're importing the class from the file instead of this module.